forge-gui/res/cardsfolder/upcoming/lumbering_megasloth.txt

```diff
@@ -0,0 +1,12 @@
+Name:Lumbering Megasloth
+ManaCost:10 G G
+Types:Creature Sloth Mutant
+PT:8/8
+K:Trample
+K:CARDNAME enters the battlefield tapped.
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {1} less to cast for each counter among players and permanents.
+SVar:X:SVar$Y/Plus.Z
+SVar:Y:Count$Valid Card.Permanent$CardCounters.ALL
+SVar:Z:PlayerCountPlayers$AllCounters
```

Comment:
needs testing but should work

Reply:
Yeah it's working now. (the Radstag as well) 

![image](https://github.com/Card-Forge/forge/assets/67333662/5345d0c3-e912-41f9-858f-e24cfd54acb0)